### DISCUSSION TITLE
欢迎来到 JSBox 讨论区!

### DISCUSSION THREAD

**Discussion by LiuGuoGY** [LiuGuoGY/JSBox-addins#8] [通用]
## 👋欢迎光临！
  我们正在使用 “Discussions” 作为与社区其他成员联系的地方。我们希望您：
  * 问您想知道的问题。
  * 分享意见。
  * 与其他社区成员互动。
  * 欢迎他人，思想开放。请记住，这是我们一起建立的社区。

**Comment by lakemonth** [comment]
小白希望有大佬能写个查快递的脚本，最好还能有小部件版🐂🍺

**Reply by LiuGuoGY** [reply]
查快递不是有吗？

**Reply by lakemonth** [reply]
🌚在erots里面没有

**Reply by LiuGuoGY** [reply]
没了，有时间我找找

**Reply by LiuGuoGY** [reply]
找过了，已经没用了，不过你可以改改试试

[Express 安装](https://xteko.com/redir?name=Express&url=https://raw.githubusercontent.com/alpha87/JsBoxScript/master/Express.js)

**Comment by tom-proxy** [comment]
大佬、求一个quanmutultX开关小组件
